src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import 'material-icons';
+import styled from 'styled-components';
+import Avatar from '@material-ui/core/Avatar';
+import SearchBar from '../SearchBar';
+import ThemeSwitch from '../ThemeSwitch/ThemeSwitch.component';
+
+const HeaderClass = styled.header`
+  display: flex;
+  width: 100%;
+  background-color: royalblue;
+  padding: 1rem 2rem;
+  justify-content: space-between;
+  align-items: center;
+`;
+const HeaderSection = styled.header`
+  display: flex;
+`;
+
```

Comment:
Si, it is ok to use semantic HTML, but if you are going to do so, you can't use `header` to contain a `header`. You can use `section` tag 👍